docs/index.md

```diff
@@ -0,0 +1,14 @@
+# StarkNet Smart Contracts
+
+## Introduction
+
+The climate emergency is the greatest challenge of our time, and requires immediate & widespread action.  
+Blockchain technology is revolutionary and opens up a new world of opportunities.  
+What if we leveraged Blockchain as a tool to fight climate change?  
+
+Our smart contracts were first deployed on Cosmos. To scale up our business we recentely decided to move on StarkNet. A blockchain's sovereignty is a tool that allows a community to generate the conditions needed to solve financial and coordination issues.
+
+- Our first criteria is the energy consumption of the blockchain that we will use. This is why we choose a blockchain with a Proof of Stake consensus.
+- Our second criteria is the maturity and resilience of the blockchain.
+- Our third criteria is the ease of access to the blockchain and a well-mature community on which CarbonABLE will develop its services.
```

Comment:
I would remove this sentence and put this one instead (a second criteria):

"Our second criteria is the scalability and security of the blockchain. To achieve unlimited scale always having the guarantee of a high level of security, our choice is the StarkNet permissionless decentralized ZK-Rollup operating as an L2 network over Ethereum’s security. "


What do you think ?